run_test.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Download .so files
+wget https://github.com/scroll-tech/da-codec/releases/download/v0.0.0-rc0-ubuntu20.04/libzktrie.so
+sudo mv libzktrie.so /usr/local/lib
+wget https://github.com/scroll-tech/da-codec/releases/download/v0.0.0-rc0-ubuntu20.04/libscroll_zstd.so
+sudo mv libscroll_zstd.so /usr/local/lib
+
+# Set the environment variable
+export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib
+
```

Comment:
we can download to this repo path then export LD_LIBRARY_PATH to contain it.
then we can avoid https://github.com/scroll-tech/go-ethereum/pull/726/files#r1595206252

Reply:
the same problem, need to assign `LDFLAGS` to the repo path, but this flag is in `da-codec` repo.

Reply:
removed.